recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,357 @@
+{% set name = "whatlies" %}
+{% set version = "0.6.5" %}
+{% set build_num = 0 %}
+{% set name_ = name|lower %}
+
+######### SET DEPENDENCY VERSION REQUIREMENTS #########
+
+{% set pyver = ">=3.6,<3.8" %}
+
+# base_packages
+{% set libver_altair = ">=4.0.1" %} 
+{% set libver_bpemb = ">=0.3.0" %}
+{% set libver_gensim = "~=3.8.3" %}
+{% set libver_matplotlib_base = ">=3.2.0" %}
+{% set libver_scikit_learn = ">=0.24.0" %}
+
+# umap_packages
+{% set libver_umap_learn = ">=0.4.0" %}
+# rasa_packages
+{% set libver_rasa = ">=2.7.0" %}
+# fasttext_packages
+{% set libver_fasttext = "~=0.9.1" %}
+# spacy_packages
+{% set libver_spacy = ">=3.0.1" %}
+{% set libver_spacy_lookups_data = ">=0.3.2" %}
+# s2v_packages (+= spacy_packages)
+{% set libver_sense2vec = ">=1.0.2" %}
+# tf_packages
+{% set libver_tensorflow = ">=2.3.0,<2.5" %}
+{% set libver_tensorflow_text = ">=2.3.0" %}
+{% set libver_tensorflow_hub = ">=0.8.0" %}
+# transformers_dep
+{% set libver_transformers = ">=4.3.0" %}
+# sentence_tfm_dep
+{% set libver_sentence_transformers = ">=0.3.8" %}
+
+#######################################################
+
+## Subpackage Build Switches
+#  This serves as a meta-level build switch 
+#  for the subpackages.
+
+{% set use_all_subpackages = True %}
+
+{% set use_umap_packages = False %}
+{% set use_rasa_packages = False %}
+{% set use_fasttext_packages = False %}
+{% set use_spacy_packages = False %}
+{% set use_s2v_packages = False %}
+{% set use_tf_packages = False %}
+{% set use_transformers_dep = False %}
+{% set use_sentence_tfm_dep = True %}
+
+#######################################################
```

Comment:
Is there a reason you're switching them on and off? I would assume that all of them should be on or otherwise they should be removed completely.

Reply:
I am using these as a top level control to selectively run certain sections of the script. Running all the builds takes close to 25 mins. However, if there is a failing build, with this setup, I can skip others and just run that one to fix it.

If you see, `use_all_subpackages` is a master switch that enforces all of them to run (overriding the individual switches, when `use_all_subpackages = True`). But when `use_all_subpackages = False`, individual switches will take precedence.